jupyterlab/commands.py

```diff
@@ -1570,7 +1579,6 @@ def _fetch_package_metadata(registry, name, logger):
     """Fetch the metadata for a package from the npm registry"""
     req = Request(
         urljoin(registry, quote(name, safe='@')),
-        method='GET',
```

Comment:
Just curious, why is this line removed?

Reply:
To answer my question, the parameter doesn't exist in the python 2 version of Request: https://docs.python.org/2/library/urllib2.html#urllib2.Request